src/test/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImplTest.java

```diff
@@ -63,6 +101,39 @@ public void tearDown() throws Exception {
         cuExtractPaymentServiceImpl = null;
         FileUtils.forceDelete(new File(ACH_BATCH_TEST_FILE_PATH).getAbsoluteFile());
     }
+    
+    @Test
+    public void testDvCheckStubCreation() {
+        int line1ArrayPosition = 0;
+        int line2ArrayPosition = 1;
+        int line3ArrayPosition = 2;
+        
+        for (int numTests = 1; numTests <= MAX_NUM_DV_CHECK_STUB_NOTES_TESTS; numTests++) {
```

Comment:
I would suggest renaming the index variable to something like "testIndex" instead, so that its intent is clearer.

Reply:
Changed